vorpal.gemspec

```diff
@@ -13,8 +13,9 @@ Gem::Specification.new do |spec|
   spec.homepage      = "https://github.com/nulogy/vorpal"
   spec.license       = "MIT"
 
-  spec.files         = `git ls-files -z`.split("\x0")
-  spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
+  spec.files         = Dir["CHANGELOG.md", "LICENSE.txt", "README.md", "vorpal.gemspec", "lib/**/*"]
+  spec.bindir        = "exe"
+  spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
```

Comment:
Should this just be `[]`, since vorpal does not expose any executables?

Just seems slightly misleading to have a "exe" bindir that doesn't actually exist. But maybe that is the common pattern.